pandas/core/generic.py

```diff
@@ -2029,7 +2049,13 @@ def to_dense(self):
 
     def __getstate__(self):
         meta = {k: getattr(self, k, None) for k in self._metadata}
-        return dict(_data=self._data, _typ=self._typ, _metadata=self._metadata, **meta)
+        return dict(
+            _data=self._data,
+            _typ=self._typ,
+            _metadata=self._metadata,
```

Comment:
are attrs linked to _metadata in any way?

Reply:
Not yet, though perhaps eventually if we can find a suitable way to deprecate `_metadata` for subclasses. With this PR, pandas doesn't use `_metadata` itself anymore (previously, we just used it for `Series.name`.

Reply:
wait, then we *should* deprecate _metadata right?

Reply:
Most likely? I have no idea who is using it or what they're using it for though.

Reply:
ok can you open an issue to deprecate that